compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/expressions/ErrorConstructorExpressionNode.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.model.tree.expressions;
+
+/**
+ * Represent error-constructor-expr.
+ *
+ * @since 2.0.0
+ */
+public interface ErrorConstructorExpressionNode extends ExpressionNode {
+
+    void addPositionalArg(ExpressionNode positionalArg);
+
+    void addNamedArgArg(NamedArgNode namedArg);
```

Comment:
Shall we add getters instead?

Arg is repeated in the second method btw.

Reply:
Added